src/components/checkboxes/_checkbox.scss

```diff
@@ -1,6 +1,6 @@
-$checkbox-input-width: 1.25rem;
-$checkbox-padding: 0.6rem;
-$checkbox-label-vertical-padding: 0.75rem;
+$checkbox-input-width: 22.5px;
+$checkbox-padding: 10.8px;
+$checkbox-label-vertical-padding: 13.5px;
```

Comment:
I don't think this is being used anywhere?

Reply:
Removed it